src/data/markdown/docs/30 xk6-browser/30 xk6-browser.md

```diff
@@ -93,22 +93,21 @@ export default function () {
-| <BWIPT /> [Request](/javascript-api/xk6-browser/request/) | [`failure()`](https://playwright.dev/docs/api/class-request#request-failure) (dependent on event-loop support in k6), [`postDataJSON()`](https://playwright.dev/docs/api/class-request#request-post-data-json), [`redirectFrom()`](https://playwright.dev/docs/api/class-request#request-redirected-from), [`redirectTo()`](https://playwright.dev/docs/api/class-request#request-redirected-to) |
-| <BWIPT /> [Response](/javascript-api/xk6-browser/response/) | [`finished()`](https://playwright.dev/docs/api/class-response#response-finished) (dependent on event-loop support in k6) |
-| [Touchscreen](/javascript-api/xk6-browser/touchscreen/) |  |
+| <BWIPT /> [Browser](/javascript-api/xk6-browser/browser/) | It is the entry point for all tests. It is used to launch `BrowserContext`s and `Page`s. |
+| <BWIPT /> [BrowserContext](/javascript-api/xk6-browser/browsercontext/) | It enables independent browser sessions, with separate `page`s, cache and cookies. |
+| <BWIPT /> [ElementHandle](/javascript-api/xk6-browser/elementhandle/) | Represents an in-page DOM element. |
+| <BWIPT /> [Frame](/javascript-api/xk6-browser/frame/) | Access and interact with the `Page`'s `Frame`s. |
+| [JSHandle](/javascript-api/xk6-browser/jshandle) | Represents an in-page JavaScript object. |
+| [Keyboard](/javascript-api/xk6-browser/keyboard/) | Used to simulate the keyboard interactions with the associated `Page`. |
+| [Mouse](/javascript-api/xk6-browser/mouse/) | Used to simulate the mouse interactions with the associated `Page`. |
+| <BWIPT /> [Page](/javascript-api/xk6-browser/page/) | `Page` provides methods to interact with a single tab in a `Browser`. |
+| <BWIPT /> [Request](/javascript-api/xk6-browser/request/) | Used to keep track of the request the `Page` makes.  |
+| <BWIPT /> [Response](/javascript-api/xk6-browser/response/) | Represents the response received by the `Page`. |
+| [Touchscreen](/javascript-api/xk6-browser/touchscreen/) | Used to simulate touch interactions with the associated `Page`. |
```

Comment:
We can provide links to these as well: Browser, Page, and Frame.

Reply:
The table contains links to these classes already. Is it going to be confusing if we add the same links in the description?

Reply:
The one I see that might not have a link is the following:

```
`Page` provides methods to interact with a single tab in a `Browser`.
```

 `Page` might not be linked because it already talks about `Page`, however, `Browser` might have a link.

In there, we could also remove `Page` as follows:

```
Provides methods to interact with a single tab in a [`Browser`](/javascript-api/xk6-browser/browser/).
```

However, for `Keyboard`, for example. It talks about `Page`. So `Page` can be a link without any confusion.

Here is another example, this one can simply say "frame" instead of "`Frame`":

```
| <BWIPT /> [Frame](/javascript-api/xk6-browser/frame/) | Access and interact with the `Page`'s frames. |
```

Reply:
Ok, sounds good.

Can there be more than one frame in a page?